src/vs/workbench/browser/parts/editor/editorCommands.ts

```diff
@@ -1368,7 +1371,8 @@ function registerOtherEditorCommands(): void {
 	});
 }
 
-function getEditorsContext(accessor: ServicesAccessor, resourceOrContext?: URI | IEditorCommandsContext, context?: IEditorCommandsContext): { editors: IEditorCommandsContext[]; groups: Array<IEditorGroup | undefined> } {
+type EditorsContext = { editors: IEditorCommandsContext[]; groups: Array<IEditorGroup | undefined> };
```

Comment:
This is a weird type to me: `editors` already have the editor group, so why is there also `groups`?